cmake/Hipify.cmake

```diff
@@ -1,6 +1,7 @@
 # cmake file to trigger hipify
 
-set(HIPIFY_SCRIPTS_DIR ${PROJECT_SOURCE_DIR})
+# Get the hipify_cli.py source directory from current directory
+get_filename_component(HIPIFY_SCRIPTS_DIR ${CMAKE_CURRENT_LIST_DIR} DIRECTORY)
```

Comment:
CMake noob question: wouldn't `set(HIPIFY_SCRIPTS_DIR ${CMAKE_CURRENT_LIST_DIR}` work?

Reply:
Consider hipify-torch is clone at 
/tensorpipe/third_party/hipify-torch
then 
`CMAKE_CURRENT_LIST_DIR = /tensorpipe/third_party/hipify-torch/cmake/Hipify.cmake` (in hipify.cmake file)
to get to parent dir, to access hipify_cli.py, we need the `get_filename_component()` which will set `HIPIFY_SCRIPTS_DIR = /tensorpipe/third_party/hipify-torch/`